### PR TITLE
prevents parent.config failure when no delivery services assinged

### PR DESCRIPTION
**Pull request by dg4prez**
Checks that there are delivery services assigned before trying to process them.

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1072/
Test PASSed.

File: traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm [hunk 1]
```diff
@@ -2465,40 +2465,43 @@ sub parent_dot_config { #fix qstring - should be ignore for quika
...
-				# This is used only if not overridden by a server profile qstring handling parameter.
-				my $ds_qsh = $qsh;
-				if (!defined($qsh)) {
-					$ds_qsh = $ds->{'param'}->{'parent.config'}->{'psel.qstring_handling'};
-				}
-				my $parent_qstring = defined($ds_qsh) ? $ds_qsh : "ignore";
 
-				if ( $ds->{qstring_ignore} == 0 && !defined($ds_qsh) ) {
-					$parent_qstring = "consider";
+		unless (!defined($data->{dslist})) {
```
Comment (dangogh) [inline traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm]: umm...   `unless (!...)` is much more easily written as `if (...)`

Comment (dg4prez) [reply traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm]: Can't really argue with that.  Updated.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1092/
Test PASSed.

**Pull request merged**